config/imsim-config.yaml

```diff
@@ -206,7 +206,7 @@ stamp:
             type: lsst_optics
             telescope: LSST
             band: { type: OpsimMeta, field: band }
-            boresight: "@input.atm_psf.boresight"
+            boresight: "@image.wcs.boresight"
```

Comment:
Oh, I didn't realize this had been purposely changed to the `"@input.atm_psf.boresight"` value before.   I can just switch it back and wait until we have a better way of providing this value.

Reply:
Wait, I see that the current `imsim-config.yaml` has the `atm_psf.boresight` value also set as 
```
boresight:  "@image.wcs.boresight"
```
So I don't see how using that same value for the `lsst_optics` photon_op would cause any problems.

Reply:
It wouldn't unless someone decides to use a different wcs for some application.  I'm just saying, this would probably be better provided by the base config in a strictly neutral place, like `eval_variables` that all other things could then use as `"$boresight"`.

Reply:
Ok.  I'll just keep it as-is then, and we can move it to `eval_variables` later.